client/state/selected-editor/actions.js

```diff
@@ -7,16 +7,16 @@
 /**
  * Internal dependencies
  */
-import { EDITOR_TYPE_REQUEST, EDITOR_TYPE_SET } from 'state/action-types';
+import { EDITOR_TYPE_FETCH, EDITOR_TYPE_UPDATE } from 'state/action-types';
 import 'state/data-layer/wpcom/sites/gutenberg';
 
 export const requestSelectedEditor = siteId => ( {
-	type: EDITOR_TYPE_REQUEST,
+	type: EDITOR_TYPE_FETCH,
```

Comment:
we have tried to eliminate the `_FETCH` terminology and prefer `_REQUEST` to express that this is a declarative request/requirement: "I need the selected editor" vs. "I need you to issue a network call"

was there a reason to change it back to `_FETCH` here?